metricbeat/module/kubernetes/event/event.go

```diff
@@ -91,14 +93,14 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 }
 
 // Run method provides the Kubernetes event watcher with a reporter with which events can be reported.
-func (m *MetricSet) Run(reporter mb.PushReporter) {
+func (m *MetricSet) Run(reporter mb.PushReporterV2) {
```

Comment:
is it required to change reporter version in this PR?

Reply:
it is a linter issue

Reply:
I reverted this